wwwapp/models.py

```diff
@@ -223,5 +225,13 @@ def is_qualified(self):
         else:
             return False
 
+    def result_in_percent(self):
+        if self.qualification_result is None:
+            return None
+        max_points = self.workshop.max_points
+        if max_points is None:
+            max_points = self.workshop.workshopparticipant_set.aggregate(max_points=models.Max('qualification_result'))['max_points']
+        return min(self.qualification_result / max_points * 100, settings.MAX_POINTS_PERCENT)
```

Comment:
Czy to nie powinno być wymuszane przy wpisywaniu wyników? Bo bez tego prowadzący może nie zauważyć